src/Lykke.Service.SwiftWithdrawal.Contracts/SwiftCashoutStateChangedEvent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MessagePack;
+
+namespace Lykke.Service.SwiftWithdrawal.Contracts
+{
+    [MessagePackObject(keyAsPropertyName: true)]
+    public class SwiftCashoutStateChangedEvent
```

Comment:
Too generic event. We should avoid such event types and use concrete, SwiftCashoutCreatedEvent as example.